TestFlashLFQ/TestFlashLFQ.cs

```diff
@@ -965,19 +965,19 @@ public static void TestBayesianProteinQuantification()
 
             var quantResult = pg.conditionToQuantificationResults["b"];
 
-            Assert.That(Math.Round(quantResult.cutoff.Value, 3) == 0.366);
-            Assert.That(Math.Round(quantResult.PosteriorErrorProbability, 3) == 0.089);
+            Assert.That(Math.Round(quantResult.NullHypothesisCutoff.Value, 3) == 0.366);
```

Comment:
there aren't (m)any comments in the flashlfq unit tests. Maybe at least add comments to these explain what is being test

Reply:
I added a comment at the top of the test